layers/sctp.go

```diff
@@ -304,19 +298,25 @@ func decodeSCTPData(data []byte, p gopacket.PacketBuilder) error {
 		StreamSequence:  binary.BigEndian.Uint16(data[10:12]),
 		PayloadProtocol: SCTPPayloadProtocol(binary.BigEndian.Uint32(data[12:16])),
 	}
+
+	if sc.PayloadProtocol == 0 {
+		sc.Payload = make([]byte, sc.SCTPChunk.Length-16)
```

Comment:
don't alloc/copy the data here, just set `sc.Payload = data[16:sc.blahblah]`.

if https://godoc.org/github.com/google/gopacket#DecodeOptions `NoCopy` is false (the default), gopacket will have already have copied this data into a new buffer.  If it's true, then the user has assumed responsibility for the passed-in buffer and will make sure it stays live.